lighthouse-cli/chrome-launcher.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const inquirer = require('inquirer');
+const net = require('net');
+const rimraf = require('rimraf');
+
+const spawn = childProcess.spawn;
+const execSync = childProcess.execSync;
+
+const platformError = new Error('Platform ' + process.platform + ' is not supported');
+
+function defaults(val, def) {
+  return typeof val === 'undefined' ? def : val;
+}
+
+module.exports = class Launcher {
+  constructor(opts) {
+    opts = opts || {};
+    // choose the first one (default)
+    this.head = defaults(opts.head, true);
```

Comment:
One approach would be to rather than describe what it is, you could describe what it does. Even something as terrible as `useFirstChromeFound` (ignoring prioritization that's actually at work there) would help make its purpose clearer